reactor-metrics-micrometer/build.gradle

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2022 VMware Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'biz.aQute.bnd.builder'
+apply plugin: 'java-library'
+
+description = 'Reactor Micrometer Metrics support'
+
+version = "$metricsMicrometerVersion"
+group = "io.projectreactor.observability"
```

Comment:
Is this group mandatory for all Reactor projects, or it is intended to be used only by reactor-core?

Reply:
only by reactor-core. I have changed the package names, module name and bnd configuration to reflect that in the module's name (now `reactor-core-micrometer`)